engine/src/main/java/org/hibernate/search/spi/impl/HashSetIndexedTypesSet.java

```diff
@@ -48,6 +49,14 @@ public boolean isEmpty() {
 		return set.stream().map( IndexedTypeIdentifier::getPojoType ).collect( Collectors.toSet() );
 	}
 
+	boolean contains(IndexedTypeIdentifier id) {
+		return set.contains( id );
+	}
+
+	Set<IndexedTypeIdentifier> cloneMap() {
```

Comment:
You mean it's a copy/paste mistake? Because the thing being cloned is not a map, either...